pkg/ebpf/ebpftest/buildmode.go

```diff
@@ -95,6 +97,23 @@ func (f fentry) Env() map[string]string {
 	}
 }
 
+type ebpfless struct{}
+
+func (e ebpfless) String() string {
+	return "eBPFless"
+}
+
+func (e ebpfless) Env() map[string]string {
+	return map[string]string{
+		"NETWORK_TRACER_FENTRY_TESTS":        "false",
+		"DD_ENABLE_RUNTIME_COMPILER":         "false",
+		"DD_ENABLE_CO_RE":                    "false",
+		"DD_ALLOW_RUNTIME_COMPILED_FALLBACK": "false",
+		"DD_ALLOW_PRECOMPILED_FALLBACK":      "false",
+		"DD_NETWORK_CONFIG_ENABLE_EBPF_LESS": "true",
```

Comment:
similar to val's comment, maybe we should also change this to `DD_NETWORK_CONFIG_ENABLE_EBPFLESS`?